EIPS/eip-663.md

```diff
@@ -1,10 +1,10 @@
 ---
 eip: 663
-title: Unlimited SWAP and DUP instructions
-description: Introduce SWAPN and DUPN which take an immediate value for the depth
-author: Alex Beregszaszi (@axic)
+title: Additional stack manipulation instructions
+description: Introduce additional stack manipulation instructions which take an immediate arguments for the depth
+author: Alex Beregszaszi (@axic), Charles Cooper (@charles-cooper), Danno Ferrin (@shemnon)
 discussions-to: https://ethereum-magicians.org/t/eip-663-unlimited-swap-and-dup-instructions/3346
-status: Review
+status: Draft
```

Comment:
We hope to get EIP-3540 out of Stagnant soon https://github.com/ethereum/EIPs/pull/8217
then this can be left in Review

Reply:
@charles-cooper :point_up: this can be undone now, 3540 is out of Stagnant.